src/components/Pages/About.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import HeroSectionAbout from '../HeroSectionAbout'
+
+const About = () => {
+    return (
+        <div>
```

Comment:
Caso for retornar apenas um componente, não é necessário colocá-lo dentro de uma _div_, caso for retornar mais de um e a _div_ seja apenas um _wrapper_, é melhor usar o fragment do React: [React Fragments](https://pt-br.reactjs.org/docs/fragments.html), é o ideal pra não precisar usar uma _div_ que possui um valor semântico.